code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -142,3 +142,20 @@
 	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridículo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porquería!")
 	leave_happy_lines = list("Amigo, era delicio. Thank you!", "Yo tuve el mono, and you friend? You hit the spot.", "Just the right amount of spicy!")
 	wait_for_food_lines = list("Ay ay ay, what's taking so long...", "Are you ready yet, amigo?")
+
+/datum/customer_data/british
+	nationality = "Space-British"
+	base_icon = "british"
+	prefix_file = "strings/names/british_prefix.txt"
+	speech_sound = 'sound/creatures/tourist/tourist_talk_british.ogg'
+	clothing_sets = list("british_gentleman", "british_bobby")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/food/fishandchips = 10, /obj/item/food/soup/stew = 10, , /obj/item/food/salad/ricepudding = 5, /obj/item/food/grilledcheese = 5, /obj/item/food/pie/meatpie = 5, /obj/item/food/benedict = 5, /obj/item/food/full_english = 2, /obj/item/food/soup/indian_curry = 3, /obj/item/food/beef_wellington/slice = 2),
```

Comment:
remove the curry from this list, it's a known fact the british don't spice their food